nvidia/advanced/canbus.rst

```diff
@@ -0,0 +1,75 @@
+How to use CAN bus
+##################
+
+The tutorial will guide you how to use CAN bus on NVIDIA Platform.
+
+Supported Model: 
+
+    * ROScube-X series
+    * ROScube-Pico series
```

Comment:
Only NX support CANbus. Nano doesn't support it.

Reply:
OK